### ISSUE TITLE
Multiline string with variable indented more on every format on php-version >= 7.3

### ISSUE THREAD

**Issue by Epskampie** [prettier/plugin-php#1593]
**@prettier/plugin-php v0.16.0**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AjAAwMA0IMAhgEblW14AsrAK7YoAFS648AM3YAbUa2zEA5nKgBFIehhR8shVFZhqYAGpJlWfCADsAOn4hWMCO3hzVAYXQBbX+wACgASgfiuQkYgnG4AxlAAyjAAnuo2YNgQALQATM4g0ACOQvDQgW4qAfryiiDEohAw5eyV7NWGAL6sUBTwMOgoaPBYUqDs2DDtol11mGBCk3ggADwA-CZgADqY27Fy49gABACC28Dbh5eH85yesYfSQpixGJiHnAAUAJQXV+dvV0BABIaPBsFkAHyxD6bEC-QEIw5iahgw6+IRyBCeTBQQ4TCCqeGIq6o4gUKAkHpoga49iYCjU6CHLCHKAANyQyTx7A59iJxMOACEFszMHIuSyslkNlkORArG8IQBeQ4OADM-OJdIZWHFhwA7tQKYd2Ic2ewCVx1IdUZAoKISHhDiD7TBNZdYV8ANxEjrbP2YEAdIA)

**Input:**
```php
<?php

class A
{
    public function b()
    {
        $this->c("
            This multiline string
            is indented more and more on every save.
            But only on --php-version >= 7.3
            and only when a variable is present: $test
        ");
    }
}

```

**Output:**
```php
<?php

class A
{
    public function b()
    {
        $this->c("
                    This multiline string
                    is indented more and more on every save.
                    But only on --php-version >= 7.3
                    and only when a variable is present: $test
                ");
    }
}

```

**Comment by kilbiller** [comment]
I have the same issue, it works on v15.0.1 but breaks from v15.0.2 and forward

**Comment by Lokdex** [comment]
Also same issue, tried downgrading to v0.15.1 as @kilbiller said, yet it still pushes the lines deeper each time I format the file.

**Comment by czosel** [comment]
Thanks for the bug report!

**Comment by czosel** [comment]
Fixed in `v0.16.1` :tada:
